### PR TITLE
Use 60 FPS for textures_image_loading example

### PR DESCRIPTION
**Pull request by nagy**
Follow up on #1816.

### CODE REVIEW

File: examples/textures/textures_image_loading.c [hunk 1]
```diff
@@ -30,6 +30,8 @@ int main(void)
     UnloadImage(image);   // Once image has been converted to texture and uploaded to VRAM, it can be unloaded from RAM
     //---------------------------------------------------------------------------------------
 
+    SetTargetFPS(60);               // Set our game to run at 60 frames-per-second
```
Comment (raysan5) [inline examples/textures/textures_image_loading.c:33]: To be consistent with all the other examples, this function call should be placed before the points-line above it.

**Review by raysan5 — COMMENTED** [review]
Please, could you check the review comment?

File: examples/textures/textures_image_loading.c (hunk 1 above)
Comment (nagy) [reply examples/textures/textures_image_loading.c]: Updated.

**Comment by raysan5** [comment]
@nagy thanks for the review!

**Pull request merged**